### ISSUE TITLE
still IndexError after 10 trails of reloading the web page

### ISSUE THREAD

**Issue by relleums** [fact-project/smart_fact_crawler#13]
It happened once this night. Seems that our approach of reloading the webpage in case of IndexError does not help. Probably we need to save the payload in such cases to have an idea of what is causing this error.

```
IndexError: list index out of range
    item = self.page_payload[row][col]
  File "/home/sebastian/Desktop/phd/smart_fact_crawler/smart_fact_crawler/table_crawler.py", line 43, in __getitem__
    self._acknowledge_error_and_wait_a_moment()
  File "/home/sebastian/Desktop/phd/smart_fact_crawler/smart_fact_crawler/table_crawler.py", line 47, in __getitem__
    'humidity': Quantity(s2f(tc[4, 1]), '%'),
  File "/home/sebastian/Desktop/phd/smart_fact_crawler/smart_fact_crawler/__init__.py", line 235, in main_page
    main_page = sfc.main_page()
  File "/home/sebastian/Desktop/phd/shifthelper/shifthelper/checks/webdim.py", line 81, in check
    self.check()
```

**Comment by MaxNoe** [comment]
See #14

**Comment by dneise** [comment]
There was a mistake in the implementation, I think. c.f. #16

**Comment by MaxNoe** [comment]
This problem persists and I think I now think I have an explanation:

In case a service is not reachable because of some reason, we uses a missing value marker, which is
either `—` (em-dash) or `---` (three dashes).

However, we often have the case where a service provides two values (Like current Median and Maximum). But smartfact only provides one missing value marker, which results in the `IndexError` because the crawler expects two columns, not just one.

**Comment by MaxNoe** [comment]
Oh, if JavaScript only had a common and well parseable data structure and format.

**Comment by MaxNoe** [comment]
I would propose the following:

`smart_fact_crawler` should never raise this `IndexError` instead return `None` for the element. 
The user can then decide whether this is an Error or not. 

For example if I request the main page to get the currents I do not care about MAGIC weather having a —.

**Comment by dneise** [comment]
At the moment this sounds nice to me, i.e. I don't see a problem with this.

However, should we log the exception or not?

**Comment by MaxNoe** [comment]
No harm in it I think. Users can set logging levels so they see those errors or not.

**Comment by MaxNoe** [comment]
We can now think about using the fallback option implemented here #23
